api/v1beta1/disruption_types.go

```diff
@@ -216,6 +224,18 @@ func (s *DisruptionSpec) validateGlobalDisruptionScope() (retErr error) {
 		}
 	}
 
+	// Rule: pulsing duration
+	if s.Pulse != nil {
+		// Pulsing disruptions should not be pulsing less than every 10 milliseconds
```

Comment:
Given we're doing the entire Inject or Clean operation between these intervals, I think we should have a much larger floor, how about 500ms, or at least 250?